app/src/ui/publish-repository/publish-repository.tsx

```diff
@@ -64,7 +65,8 @@ export class PublishRepository extends React.Component<
 
   private async fetchOrgs(account: Account) {
     const api = API.fromAccount(account)
-    const orgs = await api.fetchOrgs()
+    const orgs = [...(await api.fetchOrgs())]
```

Comment:
Syntactically, I'm not sure how I feel about this. It works, but there's a lot to unpack in this one little line.

I'll let one of the other reviewers chime in, as I think this is the only really controversial thing left...

Reply:
I think this is fine since we seem to be using ES6/TS to the fullest. The only thing I don't like very much is the `await ` keyword inside the array literal.